CustomFeeValidator.cs

```diff
@@ -92,5 +146,64 @@ static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruction> inst
                 return insts;
             }
         }
+
+        [HarmonyPatch(typeof(SaveGameManager), "Save")]

```

Comment:
I would prefer to use the UMM ModSettings framework to persist mod settings, as I do in dv-steamcutoff, and not put them into the DV save. It's a lot more straightforward and will not be prone to breaking if the DV save system changes in the future.

Reply:
✅ done